examples/command_flags.py

```diff
@@ -0,0 +1,30 @@
+from discord.ext.commands import Flag, Bot, Context
+from discord.flags import Intents
+
+intents = Intents.GUILD_MESSAGES | Intents.GUILDS
+
+bot = Bot(command_prefix='!', intents=intents)
+
+@bot.listen("on_ready")
+async def on_ready():
+    print(f"{bot.user} has logged in!")
+
+
+@bot.command()
+async def ping(ctx):
+    await ctx.send('pong!')
```

Comment:
maybe add the latency here?

```suggestion
    await ctx.send(f'pong! {round(bot.latency * 1000)}ms')
```

Reply:
i honestly just copied the command handler script as the base to start implementing but sure thing